anura/avss/client.py

```diff
@@ -14,10 +14,19 @@
     Callable,
     Generator,
     Optional,
+    TypeAlias,
 )
 
 logger = logging.getLogger(__name__)
 
+_ParsedReport: TypeAlias = (
+    AggregatedValuesReport
+    | CaptureReport
+    | HealthReport
+    | SettingsReport
+    | SnippetReport
+)
+
```

Comment:
From Python 3.12, this can also be written without imports as:
```suggestion
type _ParsedReport = (
    AggregatedValuesReport
    | CaptureReport
    | HealthReport
    | SettingsReport
    | SnippetReport
)

```
https://docs.python.org/3/library/typing.html#type-aliases

Reply:
I noticed. The fact that this is only available from 3.12 was the reason I initially decided to let the inaccurate typing persist. Maybe we ought to bump to 3.12, but lets wait until we know we won't upset anything.